source/Grid/utils/CellSizeAndPositionManager.js

```diff
@@ -225,8 +225,13 @@ export default class CellSizeAndPositionManager {
       return {};
     }
 
+    let start;
+    if (totalSize < containerSize) {
+      start = 0;
+    } else {
+      start = this._findNearestCell(offset);
+    }
```

Comment:
I'm not sure this is the correct place to fix this. `totalSize` could be estimated (eg if `CellMeasurer` is also being used) and therefor not totally accurate. It's a bit of an edge-case but I think it could introduce a problem.

Reply:
Guarding against this case is why the "scroll" handler in `Grid` has the following check:
```js
scrollLeft = Math.min(
  Math.max(0, totalColumnsWidth - width + scrollbarSize),
  scrollLeft,
);
scrollTop = Math.min(
  Math.max(0, totalRowsHeight - height + scrollbarSize),
  scrollTop,
);
```

Reply:
Yes, but this code is inside handleScrollEvent. But if everything fits into the screen, this code will never be executed.